.circleci/config-2_1.yml

```diff
@@ -866,6 +886,23 @@ commands:
         path: /tmp/cassandra/build/test/logs
         destination: logs
 
+  run_cqlshlib_tests:
+    parameters:
+      no_output_timeout:
+        type: string
+        default: 15m
+    steps:
+    - run:
+        name: Run cqlshlib Unit Tests
+        command: |
+          export PATH=$JAVA_HOME/bin:$PATH
+          time mv ~/cassandra /tmp
+          cd /tmp/cassandra/pylib
+          ./cassandra-cqlsh-tests.sh ..
+        no_output_timeout: <<parameters.no_output_timeout>>
+    - store_test_results:
+        path: /tmp/cassandra/pylib
```

Comment:
I think it is correct. That should point where you want the XML for unit tests circle runs stored. But this being a sh that moves that file around you have to [_collect_](https://circleci.com/docs/2.0/collect-test-data/#metadata-collection-in-custom-test-steps) the results instead of telling circle to _store_ them. So you give him the path where the XML is being generated iiuc. Also notice in the CI runs how this is being collected correctly

Reply:
I just felt it would be better to store them in separate log directory but ok

Reply:
That would imply changing the sh script to post the xml results file to a different folder. That folder would also not be used when tests run locally i.e. So if you don't mind I'd leave it as it is. Feel free to reopen if you change your mind and feel strongly about it.